src/lib/notification.js

```diff
@@ -7,6 +7,13 @@ const routes = {
   "Add service": "lab-dashboard-services-detail"
 }
 
+function referenceFormater(id) {
```

Comment:
can we make this as utils? in case need call this function by the other

Reply:
done mas, kindly re-review